app/native/src/screens/allBookings/AllBookingsListNode.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from 'react';
+import { createFragmentContainer, graphql } from 'react-relay';
+
+import BookingOverviewRow from '../../components/bookings/OverviewRow';
+
+import type { AllBookingsListNode as AllBookingsListNodeType } from './__generated__/AllBookingsListNode.graphql';
+import type { Navigation } from '../../types/Navigation';
+
+type Props = {
+  data: AllBookingsListNodeType,
+  navigation: Navigation,
+  showSeparator: boolean,
+};
+
+class AllBookingsListNode extends React.Component<Props> {
```

Comment:
What do you think about recompose for future? there are many component class (instead functions) only because of some handlers...

Reply:
I think there is nothing wrong with classes. We can discuss this later, not in this PR.